scripts/import_linter/import_linter.py

```diff
@@ -0,0 +1,53 @@
+import os
+import subprocess
+import multiprocessing
+import importlib
+
+git_base = subprocess.check_output(['git', 'rev-parse', '--show-toplevel']).decode().strip()
+
+
+def walk_and_ignore_pycache(directory):
```

Comment:
will this walk those swagger generated code? e.g., `src/promptflow-azure/promptflow/azure/_restclient/*`?

Reply:
Yes